src/main/resources/templates/layouts/baseLayout.html

```diff
@@ -11,7 +11,9 @@
       rel="stylesheet"
       th:href="@{{cdnUrl}/stylesheets/chs-styles.css(cdnUrl=${@environment.getProperty('cdn.url')})}"
       type="text/css"/>
-
+  <link
+      href="http://chs-dev.internal:8080/stylesheets/assets-digital-cabinet-office-gov-uk-static/govuk-template.css"
```

Comment:
This doesn't seem right at all.
We can't hard code this to the development server.

Reply:
same